src/overlay/Peer.cpp

```diff
@@ -1020,10 +1020,8 @@ Peer::recvAuth(StellarMessage const& msg)
         return;
     }
 
-    // send SCP State
-    // remove when all known peers implements the next line
```

Comment:
Is it currently the case that this comment's "when" condition has been achieved? Which protocol / release version included that change?

Reply:
this was added in 5f395e7bfd577659496c5a4a10dcfbb1e971879f (October 2017), so a long time ago